rules-tests/CodeQuality/Rector/Identical/StrlenZeroToIdenticalEmptyStringRector/Fixture/might_not_be_string.php.inc

```diff
@@ -23,8 +25,10 @@ class MightNotBeString
     public function run($value)
     {
         $empty = (string) $value === '';
-
         $empty = (string) $value === '';
+
+        $notEmpty = (string) $value !== '';
+        $notEmpty = (string) $value !== '';
```

Comment:
Can you extract these to a new fixture file? Just so it's easier to pin point later

Reply:
Done! :+1: